images/runtime/php-fpm/7.3/7.3.Dockerfile

```diff
@@ -0,0 +1,248 @@
+# From https://github.com/docker-library/php.git
```

Comment:
this file looks like an almost identical copy of the 7.2 dockerfile - can we use a template for the two?

Reply:
templating will be good, but as of now the scope is provide appservice something with a stable state. Going forward we will refactor these things as a seperate workitem